src/development/going-live/ssl.md

```diff
@@ -0,0 +1,180 @@
+# SSL in Production
```

Comment:
There are so many problems with this page. I agree it should be deleted.

Possibly there are tiny bits that could be re-used, like "Use the Platform.sh CLI to add the certificate".